### ISSUE TITLE
editing grid tables is broken

### ISSUE THREAD

**Issue by christian-korneck** [RickStrahl/MarkdownMonster#390]
In Markdown Monster 1.12 (on Win7 SP1 x64, .NET 4.7.1 32bit, OS language: English, OS locale: German), when I try to edit an existing grid table, the table in the table editor is corrupted.

#### Example:
1. I create a new table with the table editor:

![image](https://user-images.githubusercontent.com/6320032/41809545-d06a174e-76ef-11e8-9214-12b05fe3ff44.png)

2. the result looks like (which is correct):
```
+---------+------------+--------+
| Captain | Ship       | Series |
+=========+============+========+
| Kirk    | Enterprise | TOS    |
+---------+------------+--------+
| Picard  | Enterprise | TNG    |
+---------+------------+--------+
| Archer  | Enterprise | ENT    |
+---------+------------+--------+
| Sisko   | Defiant    | DS9    |
+---------+------------+--------+
| Janeway | Voyager    | VOY    |
+---------+------------+--------+
| Sulu    | Excelsior  | TOS    |
+---------+------------+--------+
```

3. when I now try to re-edit this table in the table editor (by right-clicking in the table -> `Edit Table)`, the table is corrupted:

![image](https://user-images.githubusercontent.com/6320032/41809593-4e189dd2-76f0-11e8-9b63-d4e5acface5a.png)

**Comment by RickStrahl** [comment]
Aargh. Thanks for the report... Yup an errant bracket made the table detection go off.

Fixed in: e88bba8f29eb205c122791aa84103a429460a388

**Comment by christian-korneck** [comment]
thanks!
